src/Swarm/Doc/Scenario.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- |
+-- SPDX-License-Identifier: BSD-3-Clause
+--
+-- Render a markdown document fragment
+-- from the Scenario JSON schema files.
+module Swarm.Doc.Scenario where
+
+import Control.Applicative ((<|>))
+import Control.Arrow (left, (&&&))
+import Data.Aeson
+import Data.Map.Strict qualified as M
+import Data.Maybe (fromMaybe)
+import Data.Text qualified as T
+import Swarm.Doc.Schema
+import Swarm.Util (quote, showT)
+import System.FilePath ((<.>), (</>))
+import Text.Pandoc
+import Text.Pandoc.Builder
+
+scenariosDir :: FilePath
+scenariosDir = "data/scenarios"
+
+schemasDir :: FilePath
+schemasDir = "data/schema"
+
+inputFiles :: [FilePath]
+inputFiles =
+  [ "scenario"
+  , "combustion"
+  , "display"
+  , "world"
+  ]
+
+columnHeadings :: [T.Text]
+columnHeadings =
+  [ "Key"
+  , "Default?"
+  , "Type"
+  , "Description"
+  ]
+
+makePandocTable :: (String, Schema) -> Pandoc
+makePandocTable (fn, schm) =
+  setTitle (text "JSON Schema for Scenarios") $
+    doc $
+      header 3 (text $ T.toTitle $ T.pack fn) <> para (text $ description schm) <> myTable
```

Comment:
There is a section in `Get.hs` for generating tables manually:
```
-- ----------------------------------------------------------------------------
-- GENERATE TABLES: COMMANDS, ENTITIES AND CAPABILITIES TO MARKDOWN TABLE
-- ----------------------------------------------------------------------------
```

We could reuse that here or make `Gen.hs` use Pandoc as well if Doc is split out.